### PR TITLE
Build .mo files before sdist/build_py

### PR DESCRIPTION
**Pull request by lvxnull**

### CODE REVIEW

**Comment by Stunkymonkey** [comment]
when I look at the file produced by the CI, the `locale` folder is missing. I guess you would have to adjust the CI?

**Comment by lvxnull** [comment]
I ran the same command as the CI locally and the archive had the locale directory

**Comment by Stunkymonkey** [comment]
Hmm referring to this file: https://github.com/Stunkymonkey/nautilus-open-any-terminal/actions/runs/7372377291/artifacts/1141238972

**Comment by lvxnull** [comment]
I did some further testing and this happens only when the .mo files aren't present before running the build command. `python -m build` seems to run `build_py` too late to actually capture the newly-created .mo files. I'll see if I can get it to make .mo files before `sdist` instead.

**Comment by lvxnull** [comment]
I have no idea how to get setuptools to do the right thing when compiling .mo files with a build step for both `python -m build` and `pip install .`. I reverted to previous, eager compilation instead, but I kept the `MO_FILES` glob pattern.

**Comment by Stunkymonkey** [comment]
now it looks like before.

Should we change the commands in the CI to get your previous solution working?

**Comment by lvxnull** [comment]
I managed to get it to work by hooking two different commands at once. As a side note, I hate that `pylint` requires docstrings for every class/method/function, even when they don't add any new information besides what's in the name/arguments

**Comment by Stunkymonkey** [comment]
> I managed to get it to work by hooking two different commands at once. As a side note, I hate that `pylint` requires docstrings for every class/method/function, even when they don't add any new information besides what's in the name/arguments

I am fine with disabling this. you can do this in the `pyproject.toml`

**Review by Stunkymonkey — DISMISSED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -34,4 +34,4 @@ ignore_missing_imports = true
 [tool.pylint]
 max-line-length = 120
 ignored-modules = "gi"
-disable = "global-statement,invalid-name"
+disable = "global-statement,invalid-name,C0114,C0115,C0116"
```
Comment (Stunkymonkey) [inline pyproject.toml:37]: I am more a fan of using the full names.

```suggestion
disable = "global-statement,invalid-name,missing-module-docstring,missing-class-docstring,missing-function-docstring"
```

**Review by Stunkymonkey — APPROVED** [review]

**Pull request merged**